src/wasm.cc

```diff
@@ -465,44 +491,29 @@ std::shared_ptr<WasmHandleBase> createWasm(const std::string &vm_key, const std:
         base_wasms->erase(it);
```

Comment:
This isn't a new thing, but I wonder if there is anything that would clear a failed WasmBase out of the base_wasms map.  With the local_wasms map, a fail callback is set up to clear out a map entry in case of failure ([code](https://github.com/proxy-wasm/proxy-wasm-cpp-host/blob/2cf68d6c00b4141ecf2afb5e5d51ebd21a27ed43/src/wasm.cc#L552)), but I don't see anything similar for base_wasms.

Reply:
I agree. 

But, thinking about the evolution of codes, IMHO, this kind of defensive codes seems to be good.